google/cloud/storage/internal/bucket_requests.cc

```diff
@@ -189,6 +190,55 @@ std::ostream& operator<<(std::ostream& os, PatchBucketRequest const& r) {
   return os << ", payload=" << r.payload() << "}";
 }
 
+std::ostream& operator<<(std::ostream& os, GetBucketIamPolicyRequest const& r) {
+  os << "GetBucketIamPolicyRequest={bucket_name=" << r.bucket_name();
+  r.DumpOptions(os, ", ");
+  return os << "}";
+}
+
+SetBucketIamPolicyRequest::SetBucketIamPolicyRequest(
```

Comment:
Nit: At some point, I'd like to go clean up all these single-letter variable names.  Out of scope for this commit, since they appear pretty much everywhere, but I think it increases readability to make changes like "r" -> "req", "b" -> "bindings", "m" -> "members", "p" -> "perm", etc.